app/src/lib/git/core.ts

```diff
@@ -264,6 +264,10 @@ function getDescriptionForError(error: DugiteError): string {
       return 'A lock file already exists in the repository, which blocks this operation from completing.'
     case DugiteError.NoMergeToAbort:
       return 'There is no merge in progress, so there is nothing to abort.'
+    case DugiteError.NoExistingRemoteBranch:
```

Comment:
Pretty sure I added this to Dugite, I havent gotten around to testing that it works but this PR might also close out #1325

Reply:
@say25 I'm happy to leave that open because I think it requires more context and/or a more friendly message, like what I did with #6610